### PR TITLE
MTM-44990 review of user doc for 10.14

### PR DESCRIPTION
**Pull request by andrekramer1**
Mostly clarifications based on questions on usage but some tidy of new unsubscribe subscriber feature.
Some removal of shared subscriptions mentions which (sadly still) did not make it in again.

### CODE REVIEW

File: content/reference/notifications-bundle/consumer-protocol.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ outputs:
 
 The {{< product-c8y-iot >}} Notifications 2.0 API uses a secure [WebSocket](https://en.wikipedia.org/wiki/WebSocket) to consume notifications generated by the {{< product-c8y-iot >}} API.
 
-The new endpoint is accessible only using the external {{< product-c8y-iot >}} fully qualified domain name of your {{< product-c8y-iot >}} environment and the standard SSL port 443 using a secure WebSocket connection.
+The new endpoint is accessible using the external {{< product-c8y-iot >}} fully qualified domain name of your {{< product-c8y-iot >}} environment and the standard SSL port 443 using a secure WebSocket connection. It is also available on unsecured port 80 and to microservices using "cumulocity:8111" but in most cases a secure connection is prefered.
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/consumer-protocol.md:12]: on unsecured port 80 > on the unsecured port 80 
prefered > preferred

File: content/reference/notifications-bundle/consumer-protocol.md [hunk 2]
```diff
@@ -31,13 +31,19 @@ or
 wss://your.{{< product-c8y >}}.environment.fullqualifieddomainname/notification2/consumer/?token=yourJwtTokenRequestedFromNotification2TokenService&consumer=aUniqueNameForThisConsumer
 ```
 
-There is a timeout of 5 minutes set on idle WebSocket connections after which the connection will be closed by the server side. Therefore the consumer must be prepared to handle closed connections which is required for fault tolerant operation in any case.
+### Web Socket timeouts
+
+There is a timeout of 5 minutes set on idle WebSocket connections after which the connection will be closed by the server side. Therefore the consumer must be prepared to handle closed connections which is required for fault tolerant operation in any case. All consuming microservices or applications should handle a close of the WebSocket and re-connectas necessary.
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/consumer-protocol.md:36]: should handle a close of the WebSocket and re-connectas necessary. > there is something weird in this sentence

File: content/reference/notifications-bundle/consumer-protocol.md [hunk 3]
```diff
@@ -31,13 +31,19 @@ or
...
+
+### Notification Acknowledgements
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/consumer-protocol.md:38]: Acknowledgements > acknowledgements

File: content/reference/notifications-bundle/overview.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ In the {{< product-c8y-iot >}} 10.11 release, Notifications 2.0 needs to be enab
 See the *Messaging Service - Installation & operations guide* on how to do that.
 Ingress via load balancers also needs to allow ingress for the new WebSocket endpoint that is enabled by default in {{< product-c8y-iot >}} 10.11, which might affect custom deployments that manage their own ingress.
 
-To receive notifications over the 2.0 protocol, an application or microservice must subscribe to notifications, either for notifications about a particular managed object or in a wider context that is scoped to a tenant.
+To receive notifications over the 2.0 protocol, an application or microservice must subscribe to notifications, either for notifications about a particular managed object or in a wider context that is scoped to a tenant. Subscriptions are persitent and long lived and allow notifications to be stored reliably until consumed and acknowledged by a consuming microservice or application.
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/overview.md:18]: persitent > persistent
are persitent and long lived and allow > are persistent, and long lived, and allow

File: content/reference/notifications-bundle/overview.md [hunk 5]
```diff
@@ -55,9 +57,11 @@ See the [{{< openapi >}}](https://{{<domain-c8y>}}/api/{{< c8y-current-version >
 For the protocol consumer, both managed object creations and alarms subscribed under the tenant context are reported in the same way.
 There is no distinction between the two contexts for consumers, and notification ordering is maintained between the two contexts.
 
-For Java developers, the API and the protocol have been wrapped up as an open Java API and a sample WebSocket client application.
+### Building consuming microservices and applications
+
+For Java developers, the API and the protocol have been wrapped up as an open Java API and a sample WebSocket client application. Any WebSocket library or programming language can be used as the protocol is text based and relatively simple. Consumer can be either microservices or applications running externally from Cumulocity and require only a JWT string when connecting.
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/overview.md:62]: text based > text-based
from Cumulocity > from {{< product-c8y-iot >}

File: content/reference/notifications-bundle/overview.md [hunk 6]
```diff
@@ -70,21 +74,27 @@ As the token string is a JWT (JSON Web Token), it can be decoded to extract the
...
+
+Once a subscription is made, notifications will be retained until consumed by all subscribers who have previously connected to the subscription.
+The normal workflow is to delete subscriptions when no longer interested in notifications and this is the resonsibility of the subscriber. 
+The subscription API [{{< openapi >}}](https://{{<domain-c8y>}}/api/{{< c8y-current-version >}}/#tag/Subscriptions) is used to delete subscriptions.
+After the subscription is deleted no more notifications will be saved. The consuming microservice or application can then drain down notifications 
+and be removed when that is done.
 
-Once a subscription is made, notifications will be kept until consumed by all subscribers who have previously connected to the subscription.
-For non-volatile subscriptions, this can result in notifications remaining in storage if never consumed by the application.
+However, unconsumed notifications will be retained, and for high throughput scenarios, this can result in notifications remaining in storage
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/overview.md:85]: remove comma after scenarios

File: content/reference/notifications-bundle/consumer-protocol.md (hunk 2 above)
Comment (andrekramer1) [reply content/reference/notifications-bundle/consumer-protocol.md:36]: Suggest: Therefore the consumer must be prepared to handle closed connections which is required for fault tolerant operation in any case.

Comment (BeateRixen) [reply content/reference/notifications-bundle/consumer-protocol.md]: The suggestion is to drop the last sentence? Ok for me.

**Review by BeateRixen — APPROVED** [review]

**Review by gapa-c8y — APPROVED** [review]

**Pull request merged**